NomadicNibbler/foodtruck-fe
# Feature/truck details

## What does this PR do?

- The truck details component was built to display extra information about each food truck
- The truck markers on the MapView component now link to the truck card details
- The truck cards on the TruckList component are now lined to the truck card details
- Testing was added for the display of the truck card details
    - Code was added the beginning of the testing file to unregister the service worker when the test suite runs

## How should it be tested?

- Login to the application using the username "test"
- click on a truck icon, the truck details should display
- navigate back to the map and click on the truck list button and select a card to click
- After clicking, the truck details should again display
- Run the cypress test suite, all tests should be passing

## What do the future iterations hold?
- Additional styling/formatting
- Better responsive styling
- Potentially display more information
## Linked Issues
- #40
